### PR TITLE
Add virtualenvwrapper support to docs

### PR DESCRIPTION
**Pull request by luabud**
@brettcannon can you review this please? 😊

### CODE REVIEW

**Comment by gregvanl** [comment]
Adding `don't merge` label as this PR is waiting on a review from @brettcannon

**Comment by luabud** [comment]
thanks!

File: docs/python/unit-testing.md [hunk 1]
```diff
@@ -244,13 +244,14 @@ Support for running tests in parallel with pytest is available through the `pyte
    python3 -m pip install pytest-xdist
    ```
 
-1. Next, open the workspace `settings.json` file and add `"-n <number of CPUs>"` to the pytest arguments list, specifying the number of CPUs you'd like to use.
+2. Next, create a file named ```pytest.ini``` in your project directory and add the content below, specifying the number of CPUs to be used. For example, to set it up for 4 CPUs:
```
Comment (brettcannon) [inline docs/python/unit-testing.md]:
```suggestion
2. Next, create a file named `pytest.ini` in your project directory and add the content below, specifying the number of CPUs to be used. For example, to set it up for 4 CPUs:
```

File: docs/python/unit-testing.md [hunk 2]
```diff
@@ -244,13 +244,14 @@ Support for running tests in parallel with pytest is available through the `pyte
...
 
-   ```json
-   "python.testing.pytestArgs": ["-n 4"]
+   ```
```
Comment (brettcannon) [inline docs/python/unit-testing.md]:
```suggestion
   ```ini
```

**Comment by brettcannon** [comment]
@gregvanl LGTM

**Comment by gregvanl** [comment]
Published. Nice work!

**Pull request merged**